app/managers.py

```diff
@@ -0,0 +1,38 @@
+from models import Actor
+
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
The database connection is opened in the constructor but never closed. Consider implementing a method to close the connection to prevent resource leaks.